libs/agno/agno/agent/agent.py

```diff
@@ -760,6 +769,15 @@ def _run(
         # 3. Update the RunOutput with the model response
         self._update_run_response(model_response=model_response, run_response=run_response, run_messages=run_messages)
 
+        if self.store_media:
+            self._store_media(run_response, model_response)
+        else:
+            if run_response.input is not None:
```

Comment:
Also need to scrub messages, etc? 

Also `run_response.images`? Or will that be skipped if you don't call `_store_media`?